sklearn/ensemble/_stacking.py

```diff
@@ -955,7 +956,7 @@ class StackingRegressor(RegressorMixin, _BaseStacking):
         .. versionadded:: 1.0
 
     final_estimator_ : estimator
-        The regressor to stacked the base estimators fitted.
+        The regressor to stack the base estimators fitted.
```

Comment:
```suggestion
        The fitted estimator on top of fitted stacking estimators.
```
Maybe this is more clear to read?

Reply:
Looking at:

https://github.com/scikit-learn/scikit-learn/blob/f6f6b776f01e12591268371c8943450179fe0651/sklearn/ensemble/_stacking.py#L560-L561

I changed to: "The final regressor, fitted on the output of `estimators_`." WDYT...?